src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -114,11 +114,14 @@ internal sealed class TaggerProvider : IViewTaggerProvider, ITableDataSource, II
             {
                 lock (taggers)
                 {
-                    if (!taggers.ExistsForFile(textDocument.FilePath))
+                    IssueTagger tagger = null;
+                    // Always return the same tagger for a given document.
+                    // If we don't, then we won't automatically get support for tooltips for issues.
+                    if (!taggers.TryGetValue(textDocument.FilePath, out tagger))
```

Comment:
The built-in _QuickInfoSource_ implementation makes a separate request for a tagger i.e. there are (at least) two request for a tagger for a given document/text buffer, from different sources. Previously we were only returning a tagger for the initial request, which was enough to make squigglies appear under issues, but not enough to have tooltips appearing too.
Strictly speaking, we should be returning separate tagger instances for each request since in theory they could be disposed of separately. However, it seems that none of the tagger-requesting callers are disposed until the document is closed. I'll try some more manually testing to see if I can get it to break (I suspect opening a second tab for the same document might do it).